package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-localize",
-  "version": "5.0.1",
+  "version": "5.1.0",
```

Comment:
Does the hook addition mean we need to bump up the React peerDependency suggestion? Or is it something that will be polyfilled down on older versions?

Reply:
`Hook` was added on version `5.0.0`, and at that time I added warning regarding using `Hooks` with older versions, I think we are good in that sense.

That said, will approve and merge @yankouskia's PR.